### ISSUE TITLE
Catboost taking too much time and memory

### ISSUE THREAD

**Issue by Prabhat-IIT** [catboost/catboost#395]
Problem: I have a training data of around 67 lakh rows and 22 columns on which I have done 90-10 train-test spilt. Now when I'm training catboost classifier on it with 0.05 learning rate it is taking around 6.5 hrs to complete 500 iterations and consuming memory of around 25 GB. I don't know what's wrong with it. Lightgbm on the other hand is taking less than 0.5 hr and much less memory in the same setting

catboost version: {0.8.1}

Operating System: {NAME="Ubuntu"
VERSION="16.04.4 LTS (Xenial Xerus)"}

CPU: {-cpu:0                 
       description: CPU
       product: Intel(R) Xeon(R) CPU E5-2686 v4 @ 2.30GHz
       vendor: Intel Corp.
       physical id: 401
       bus info: cpu@0
       slot: CPU 1
       size: 2699MHz
       capacity: 3GHz
       width: 64 bits
}

**Comment by annaveronika** [comment]
@Prabhat-IIT  is it possible for you to share the dataset with us?
Also what is the mode you are training? What are the parameters? Does your dataset contain sparse data?

**Comment by annaveronika** [comment]
closing because of inactivity
